save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationAdminView.kt

```diff
@@ -203,7 +203,13 @@ internal class OrganizationAdminView : AbstractView<Props, OrganizationAdminStat
         state.organizations = mutableListOf()
     }
 
-    private fun localButtonAction(organization: OrganizationDto, newStatus: OrganizationStatus) {
+    /**
+     * This function delete [organization] by [organizations], add [organization] with [newStatus] in [organizations] and sorted its by comparator
+     *
+     * @param organization
+     * @param newStatus
+     */
+    private fun updateOrganizationStatusInOrganizationDtoList(organization: OrganizationDto, newStatus: OrganizationStatus) {
         setState {
             organizations = organizations.minus(organization).plus(organization.copy(status = newStatus)).sortedWith(comparator)
         }
```

Comment:
This function seems to be very similar to the one for `Project`. Maybe make it be generic?

Reply:
I do not think that a generic is needed to combine 2 functions, if more appear, I will redo it.